### PR TITLE
docs(input.md): add locator API references to `Checkboxes and radio buttons` section in `input.md`

### PR DESCRIPTION
**Pull request by azad-derakhshani-GS**
This adds the missing API references mentioned in #13561.

### CODE REVIEW

File: docs/src/input.md [hunk 1]
```diff
@@ -184,6 +184,9 @@ await page.CheckAsync("text=XL");
 - [`method: ElementHandle.check`]
 - [`method: ElementHandle.isChecked`]
 - [`method: ElementHandle.uncheck`]
```
Comment (rwoll) [inline docs/src/input.md:186]:
```suggestion
```

Let's remove the ElementHandle variants while you are here as Locators are best-practice in most use-cases.

**Review by rwoll — CHANGES_REQUESTED** [review]
Thanks for your contribution! We appreciate it!

I've left one minor comment (and there should be a button in the GH UI to accept it).

CLA is signed, so looks good minus the nit!

Cheers! 🎉

**Comment by azad-derakhshani-GS** [comment]
Yeah sure why not! 
Just committed your suggestion. :)

**Review by rwoll — APPROVED** [review]

**Comment by rwoll** [comment]
Merged! Thanks @azad-derakhshani-GS!

**Pull request merged**